src/lib/Microsoft.Health.Events/EventCheckpointing/StorageCheckpointClient.cs

```diff
@@ -74,6 +75,8 @@ public async Task UpdateCheckpointAsync(Checkpoint checkpoint)
 
             try
             {
+                ct.ThrowIfCancellationRequested();
+
                 try
                 {
                     await blobClient.SetMetadataAsync(metadata);
```

Comment:
Continue passing the token into the below async methods. This will give them the chance to stop operations on a cancellation request.

Reply:
yep thanks